File: VB/Solution28.Module.Web/Controllers/MyProcessActionContainerHolderController.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.ExpressApp.Actions
Imports DevExpress.ExpressApp.Web.SystemModule
Imports DevExpress.ExpressApp.Web.Templates.ActionContainers
Imports DevExpress.ExpressApp.Web.Templates.ActionContainers.Menu

Namespace Solution28.Module.Web.Controllers
    Public Class MyProcessActionContainerHolderController
        Inherits ProcessActionContainerHolderController

        Protected Overrides Function OnCreateCustomMenuActionItem(ByVal action As ActionBase) As MenuActionItemBase
            If action.Id = "FilterRange" Then
                Return New ParametrizedRangeActionMenuActionItem(CType(action, ParametrizedAction))
            End If
            Return Nothing
        End Function
    End Class
End Namespace

File: CS/Solution28.Module.Web/Controllers/MyProcessActionContainerHolderController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.ExpressApp.Actions;
using DevExpress.ExpressApp.Web.SystemModule;
using DevExpress.ExpressApp.Web.Templates.ActionContainers;
using DevExpress.ExpressApp.Web.Templates.ActionContainers.Menu;

namespace Solution28.Module.Web.Controllers {
    public class MyProcessActionContainerHolderController : ProcessActionContainerHolderController {
        protected override MenuActionItemBase OnCreateCustomMenuActionItem(ActionBase action) {
            if(action.Id == "FilterRange") {
                return new ParametrizedRangeActionMenuActionItem((ParametrizedAction)action);
            }
            return null;
        }
    }
}
